spec/models/user_CRUD_posts_spec.rb

```diff
@@ -1,106 +1,113 @@
-    fill_in('Link', with: title_one.link)
-
-    expect(page).to have_content('EastWillyB')
-    expect(page).to have_content('Willie Jr.’s got his work cut out for him.')
+describe "posts model" do
+  before(:each) do
+    @title_one = Post.create(webseries_name: "EastWillyB", link_to_trailer: "https://www.youtube.com/watch?v=lMg2Fv-8FsM", description: "Willie Jr.’s got his work cut out for him.", link: "http://www.eastwillyb.com/")
+    @title_two = Post.create(webseries_name: "Ylse: A Web Series", link_to_trailer: "https://www.youtube.com/watch?v=h205FN_fwWs", description: "The Funniest TV show not on television", link: "http://www.ylse.net")
+    @title_three = Post.create(webseries_name: "Valley Meadows", link_to_trailer: "https://www.youtube.com/watch?v=TPoFCokBC5U", description: "Two dudes try to rap", link: "https://www.youtube.com/watch?v=q5R2Iy-5EUA")
   end
 
-  scenario "User adds item unsuccessfully" do
-    visit '/'
-    click_button('Post a Series')
-    fill_in('Title', with: title_one.title)
-    fill_in('Media Type', with: title_one.media_type)
-    fill_in('Description', with: title_one.description)
-
-    expect(page).to_not have('EastWillyB')
-    expect(page).to_not have('Willie Jr.’s got his work cut out for him.')
-  end
-
-  scenario "User adds item unsuccessfully" do
-    visit '/'
-    click_button('Post a Series')
-    fill_in('Media Type', with: title_one.media_type)
-    fill_in('Description', with: title_one.description)
-    fill_in('Link', with: title_one.link)
-
-    expect(page).to_not have('http://www.eastwillyb.com/')
-    expect(page).to_not have('Willie Jr.’s got his work cut out for him.')
+  feature "As a user, I want to add an item, So that others can review it" do
+
+    scenario "User adds item successfully" do
+      visit '/'
+      click_button('Post a Series')
+      fill_in('Title', with: @title_one.webseries_name)
+      fill_in('Link to Trailer', with: @title_one.link_to_trailer)
+      fill_in('Description', with: @title_one.description)
+      fill_in('Link to Webseries', with: @title_one.link)
+      click_button('Submit')
+
+      expect(page).to have_content('EastWillyB')
+      expect(page).to have_content('Willie Jr.’s got his work cut out for him.')
+    end
+
+    scenario "User adds item unsuccessfully" do
+      visit '/'
+      click_button('Post a Series')
+      fill_in('Title', with: @title_one.webseries_name)
+      fill_in('Link to Trailer', with: @title_one.link_to_trailer)
+      fill_in('Description', with: @title_one.description)
+      click_button('Submit')
+
+      expect(page).to_not have_content('EastWillyB')
+      expect(page).to_not have_content('Willie Jr.’s got his work cut out for him.')
+    end
+
+    scenario "User adds item unsuccessfully" do
+      visit '/'
+      click_button('Post a Series')
+      fill_in('Link to Trailer', with: @title_one.link_to_trailer)
+      fill_in('Description', with: @title_one.description)
+
```

Comment:
Try to stay consistent with spacing between features and assertions in your tests. Also, fill-ins could be differentiated with spacing as well, like so:
click thing

fill_in thing
fill_in another thing

click another thing

but that's just a personal preference! Not 100% necessary :D